packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -49,18 +69,31 @@ class PopupMenuItem<T> extends StatelessComponent {
         decoration: !hasDivider ? null : new BoxDecoration(
           border: new Border(bottom: new BorderSide(color: theme.dividerColor))
         ),
-        child: new DefaultTextStyle(
-          style: style,
-          child: new Baseline(
-            baseline: _kMenuItemHeight - _kBaselineOffsetFromBottom,
-            child: child
-          )
-        )
+        child: item
       )
     );
   }
 }
 
+class CheckedPopupMenuItem<T> extends PopupMenuItem<T> {
+  CheckedPopupMenuItem({
+    Key key,
+    T value,
+    isChecked: false,
```

Comment:
Yes, I'm going to remove all of the "is" prefixes I've added in the last 24 hours or so :-).